docs/pages/blog/lab-date-pickers-to-mui-x.md

```diff
@@ -62,7 +62,7 @@ All other date and time picker components are available in `mui/x-date-pickers`
 
 ## How do I purchase a commercial license?
 
-Visit [the MUI Store](/store/items/material-ui-pro/) to purchase a commercial license.
+Visit [the MUI Store](https://mui.com/store/items/material-ui-pro/) to purchase a commercial license.
```

Comment:
This fulfillment URL will likely change in the future. Once we start to work on v6 this link will be present in https://v5.mui.com/blog/lab-date-pickers-to-mui-x/, it will be painful to fix. Better if we can relay on the 301 that https://mui.com/store/items/material-ui-pro/ will have.